jsk_naoqi_robot/naoqieus/naoqi-interface.l

```diff
@@ -213,5 +213,40 @@
 	     (setq res (ros::service-call (format nil "~A/pose/life/getState" naoqi-namespace) req))
 	     (send res :status :data)
 	     )
+  (:get-move-arms-enabled
+   (&optional (arm :arms))
+   (let ((ret (instance naoqi_bridge_msgs::GetArmsEnabledRequest :init))
+	 (arm_msg (instance naoqi_bridge_msgs::Arms :init)))
```

Comment:
Thank you very much for your review.
```res``` is modified to be defined as a local variable.